search-service/src/main/kotlin/com/egm/stellio/search/service/TemporalEntityService.kt

```diff
@@ -7,10 +7,18 @@ import org.springframework.stereotype.Service
 import java.net.URI
 
 typealias SimplifiedTemporalAttribute = Map<String, Any>
+typealias QueryTemporalEntitiesResult = List<Pair<URI, Map<TemporalEntityAttribute, List<AttributeInstanceResult>>>>
```

Comment:
a interesting typealias would be `Map<TemporalEntityAttribute, List<AttributeInstanceResult>>` as it is your common structure between the two buildXXX functions

Reply:
Done